mycity/mycity/intents/voting_intent.py

```diff
@@ -0,0 +1,38 @@
+"""
+Functions for voting information including polling location information
+"""
+
+from . import intent_constants
+from streetaddress import StreetAddressParser
+from mycity.mycity_response_data_model import MyCityResponseDataModel
+import mycity.utilities.arcgis_utils as arcgis_utils
+import requests
+import logging
+
+logger = logging.getLogger(__name__)
+CARD_TITLE = "Polling Location"
```

Comment:
The name of this intent is `voting_intent` but the card title is `"Polling Location"`. Maybe change the name of this intent?